update.sh

```diff
@@ -67,18 +67,16 @@ function update_node_version() {
     # Add GPG keys
     for key_type in "node" "yarn"; do
       while read -r line; do
-        pattern="\"\\$\\{$(echo "$key_type" | tr '[:lower:]' '[:upper:]')_KEYS\\[@\\]\\}\""
-        sed -E -i.bak -e "s/([ \\t]*)($pattern)/\\1${line}${new_line}\\1\\2/" "$dockerfile" && rm "$dockerfile".bak
+        pattern="\"\\$\\{$(echo "${key_type}" | tr '[:lower:]' '[:upper:]')_KEYS\\[@\\]\\}\""
+        sed -Ei -e "s/([ \\t]*)(${pattern})/\\1${line}${new_line}\\1\\2/" "${dockerfile}"
       done <"keys/$key_type.keys"
-      sed -E -i.bak "/$pattern/d" "$dockerfile" && rm "$dockerfile".bak
+      sed -Ei -e "/${pattern}/d" "${dockerfile}"
     done
 
-    if [ "${version/.*/}" -ge 10 ]; then
-      sed -E -i.bak 's/FROM (.*)alpine:3.4/FROM \1alpine:3.7/' "$dockerfile"
-      rm "$dockerfile.bak"
-    elif [ "${version/.*/}" -ge 8 ] || [ "$arch" = "ppc64le" ] || [ "$arch" = "s390x" ] || [ "$arch" == "arm64" ] || [ "$arch" == "arm32v7" ]; then
-      sed -E -i.bak 's/FROM (.*)alpine:3.4/FROM \1alpine:3.6/' "$dockerfile"
-      rm "$dockerfile.bak"
+    if [[ "${version/.*/}" -ge 10 ]]; then
+      sed -Ei -e 's/FROM (.*)alpine:3.4/FROM \1alpine:3.7/' "${dockerfile}"
+    elif [[ "${version/.*/}" -ge 8 || "$arch" == "ppc64le" || "$arch" == "s390x" || "$arch" == "arm64" || "$arch" == "arm32v7" ]]; then
+      sed -Ei -e 's/FROM (.*)alpine:3.4/FROM \1alpine:3.6/' "${dockerfile}"
```

Comment:
Can we keep using `[ ]` instead of `[[ ]]` here?

Reply:
Oh my bad! rebase problem!